### ISSUE TITLE
[gnoweb] View the source of a package in Gno Playground with "." shortcut

### ISSUE THREAD

**Issue by 0xtekgrinder** [gnolang/gno#2762]
## Description

Similar to github, when pressing "." key or accessing to gno.land.dev of a package / realm, launch an instance of gno playground with the files of the package or the realm respectively.

It can ease the browsing and let users the ability to interact with them from a web interface.

**Comment by leohhhn** [comment]
Cool suggestion. We'll take it into consideration. 

cc @salmad3 @ilgooz @alexiscolin

**Comment by alexiscolin** [comment]
Yeah these kind of UX improvements and "shortcuts" are very interesting!

**Comment by leohhhn** [comment]
Would implement this.

**Comment by L33TSP34KER** [comment]
I’d like to work on this issue.

My approach is to send a request to:
https://grpc.infra.gnosh.dev/gnostudio.services.playground.v1beta.PlaygroundService/CreateSession

with the contents of the files, and then automatically redirect the user using the response.
